src/data/PhasesConst.ts

```diff
@@ -0,0 +1,80 @@
+import { useTranslation } from "react-i18next";
+
+export interface Phase {
+  title: string;
+  description: string;
+  symptoms: string[];
+}
+
+export function Phases(idx: number): Phase {
+  const { t } = useTranslation();
+
+  const phases: Phase[] = [
```

Comment:
`phases` is const and will be created at every time when you calling this function

This module is fully isolated and you can create this object just once in the module scope